### PR TITLE
Redirect to spotify button

### PR DESCRIPTION
**Pull request by eripheebs**

### CODE REVIEW

File: frontend/www/js/controllers/goToSpotifyController.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        scheme = 'spotify://';
+    } else if(device.platform === 'Android') {
+        scheme = 'com.spotify.android';
+    };
+
+    appAvailability.check(
+      scheme,
+      function() {
+          window.open('spotify://', '_system', 'location=no');
+          console.log('Twitter is available');
```
Comment (robota-x) [inline frontend/www/js/controllers/goToSpotifyController.js:22]: not sure on the function of this. intended?

File: frontend/www/js/controllers/goToSpotifyController.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+playfestApp.controller('goToSpotifyController', function($scope, $cordovaInAppBrowser) {
+
+  document.addEventListener("deviceready", onDeviceReady, false);
+
+  function onDeviceReady() {
+    console.log(device.cordova);
```
Comment (robota-x) [inline frontend/www/js/controllers/goToSpotifyController.js:6]: debugging code?

**Comment by robota-x** [comment]
👍 :shipit:

File: frontend/www/js/controllers/goToSpotifyController.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  };
+
+  $scope.launchSpotify = function(){
+    var scheme;
+
+    if(device.platform === 'iOS') {
```
Comment (JojSh) [inline frontend/www/js/controllers/goToSpotifyController.js:12]: Could refactor this if/else if as a ternary?

**Pull request merged**